src/services/request.service.js

```diff
@@ -27,4 +27,25 @@ export default class RequestService {
       }
     );
   }
+
+  /**
+   * @returns {Integer} returns number of traveled places
+   * @description returns all of the destinations a and their appearance times from the database
+   */
+  static getMostTraveledDestinations = async () => {
+    const placeAndTheirVisitTimes = await sequelize.query('SELECT "travelTo", COUNT(*) FROM requests WHERE status !=\'pending\' AND status!=\'rejected\' GROUP BY "travelTo" ORDER BY count DESC');
```

Comment:
Good job @de-scholar but I think that here you should use where status= 'approved' cause currently when I change the status to something which is not `pending` or `rejected` like `sdabdhbfbfd` in my database it lists that trip on that destinations list you are returning.

Reply:
I changed this